daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -148,19 +148,26 @@ def enable(name=None, verbose=True, deprecation=True):
             do_patch(key)
     if deprecation:
         set_idp_sklearn_verbose()
-        warn("Scikit-learn patching with daal4py is deprecated "
-             "and will be removed in the future.\n"
-             "Please, use Intel(R) Extension for Scikit-learn module instead "
-             "(pip install scikit-learn-intelex)\n"
-             "To enable patching, please, use one of options:\n"
-             "1) python -m sklearnex <your_script>\n"
-             "2) from sklearnex import patch_sklearn\n"
-             "   patch_sklearn()",
-             FutureWarning, stacklevel=2)
-    if verbose and sys.stderr is not None:
+        warnings.warn_explicit("\nScikit-learn patching with daal4py is deprecated "
+                               "and will be removed in the future.\n"
+                               "Use Intel(R) Extension "
+                               "for Scikit-learn* module instead "
+                               "(pip install scikit-learn-intelex).\n"
+                               "To enable patching, please use one of the "
+                               "following options:\n"
+                               "1) python -m sklearnex <your_script>\n"
+                               "2) Enable patching from your script:"
```

Comment:
maybe something like this is better (more consistent)

```suggestion
                               "1) From the command line:\n"
                               "    python -m sklearnex <your_script>\n"
                               "2) From your script:\n"
```